Cargo.toml

```diff
@@ -18,7 +18,7 @@ libc = "0.2"
 [target.'cfg(windows)'.dependencies]
 crossbeam = "0.2.10"
 kernel32-sys = "0.2.2"
```

Comment:
kernel32-sys has been merged into winapi 0.3, so this dependency is not needed anymore. :)